{{cookiecutter.repostory_name}}/app/Dockerfile

```diff
@@ -1,19 +1,38 @@
-FROM python:3.6
+{% if cookiecutter.use_alpine_linux == "y" -%}
+ARG BASE_IMAGE=python:3.6-alpine
+{% else -%}
+ARG BASE_IMAGE=python:3.6
+{% endif -%}
 
-COPY ./bin/s6-overlay-amd64.tar.gz /tmp/s6-overlay-amd64.tar.gz
+FROM $BASE_IMAGE AS base-image
 
-RUN tar xvfz /tmp/s6-overlay-amd64.tar.gz -C / \
-    && rm -f /tmp/s6-overlay-amd64.tar.gz {% if cookiecutter.use_celery == "y" %}\
-    && adduser --disabled-password --gecos '' celery{% endif %}
+{% if cookiecutter.use_alpine_linux == "y" -%}
+RUN apk add postgresql-dev gcc python3-dev musl-dev
+{% endif -%}
 
-COPY ./src/ /srv/app/
+ENV PYTHONUNBUFFERED=1
+ENV PATH=/root/.local/bin:$PATH
 
-WORKDIR /srv/app
+RUN pip3 install --user --no-cache-dir psycopg2-binary==2.8.3 gunicorn==19.9.0
```

Comment:
hardcoded versions. Grep them out from `requirements.txt`

Reply:
So, `gunicorn` hasn't been defined in `requirements.txt` and was never pinned to any version. I also removed `psycopg2` from `requirements.txt` and I don't know if it really needs to exists in there.

Reply:
we may not need postgres at all if we are setting up a project for a customer that uses mysql or mssql (it sometimes happens)